pandas/tests/series/test_duplicates.py

```diff
@@ -59,12 +59,19 @@ def test_unique_data_ownership():
     Series(Series(["a", "c", "b"]).unique()).sort_values()
 
 
-def test_is_unique():
-    # GH11946
-    s = Series(np.random.randint(0, 10, size=1000))
-    assert s.is_unique is False
-    s = Series(np.arange(1000))
-    assert s.is_unique is True
+@pytest.mark.parametrize('data, expected', [
```

Comment:
do we have sufficient tests on the Index side for this? (e.g. your parameterization is good, maybe add for the index tests)/